src/hooks/createRequest.js

```diff
@@ -1,8 +1,8 @@
 
 export default function createRequestObject(type, filter) {
 
-	const api = 'https://hdwj2rvqkb.us-west-2.awsapprunner.com'
-	// const api = 'http://127.0.0.1:8086'
+	// const api = 'https://hdwj2rvqkb.us-west-2.awsapprunner.com'
+	const api = 'http://127.0.0.1:8080'
```

Comment:
don't forget to change this

Reply:
> 

yea footlocker is kinda rare - one sneaker that has results is the Jordan Dub Zero in White/Cool grey ([here](https://www.footlocker.ca/en/product/jordan-dub-zero-mens/31046107.html))